src/System Application/App/Telemetry/src/Logging/TelemetryImpl.Codeunit.al

```diff
@@ -34,15 +34,33 @@ codeunit 8712 "Telemetry Impl."
     var
         TelemetryLoggers: Codeunit "Telemetry Loggers";
         TelemetryLogger: Interface "Telemetry Logger";
+        RelevantTelemetryLoggers: List of [Interface "Telemetry Logger"];
     begin
         AddCustomDimensionsFromSubscribers(CustomDimensions, Publisher);
         AddCustomDimensionsSafely(CustomDimensions, CallerCustomDimensions);
 
         TelemetryLoggers.SetCurrentPublisher(Publisher);
```

Comment:
Small performance improvement can be made here:
If we add something like `TelemetryLoggers.SetCurrentTelemetryScope`, then we don't need to collect other extension's loggers when scope is set to `ExtensionPublisher`.